src/main/java/core/turnorders/AlternatingTurnOrder.java

```diff
@@ -21,6 +21,15 @@ protected TurnOrder _copy() {
         return to;
     }
 
+    @Override
+    public void _endRound(AbstractGameState gameState) {
```

Comment:
I like these methods, but I don't like that all of these methods are implemented in the subclasses doing nothing. Should not be forced.

Reply:
Do you mean 'should be forced'?

I am also torn between making these abstract (and hence forcing an implementor to make a decision); and having a dummy implementation in some concrete parent class to avoid code boilerplate and stub methods that don't do anything and clutter up implementation classes.

Reply:
But this dummy implementation could just be on the abstract turn order, instead of cluttering all the other subclasses.